client/src/test/java/io/pravega/client/state/impl/RevisionedStreamClientTest.java

```diff
@@ -58,18 +58,15 @@
 import java.util.Arrays;
 import java.util.Iterator;
 import java.util.Map.Entry;
+import java.util.NoSuchElementException;
 import java.util.TreeMap;
 import java.util.concurrent.CompletableFuture;
 import lombok.Cleanup;
 import org.junit.Test;
 import org.mockito.Mockito;
 
 import static io.pravega.test.common.AssertExtensions.assertThrows;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
please don't use `import *`

Reply:
This is interesting point @shshashwat , I know in eclipse we have some setting which will avoid using * for any import and we can configure. Here we ran ./gradlew clean checkstyleMain checkstyleTest but this is not pointed about this style. So do we need to update out checkstyle configuration for this ?

Reply:
It's not caught because it's a static import of methods. Which isn't really an issue. I would still configure exclipse / idea to not do that so as to avoid needlessly changing the formatting of existing code.